### PR TITLE
Refactor README for 1.0

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*
#3776

*Description of changes:*
Improve and update the README file with updated links and improved wording.

Quick link to the updated README: https://github.com/Innixma/autogluon/blob/update_readme_1.0/README.md

TODO:

- [x] Center Image, Title, Badges, Opener
- [ ] Change the tagline, don't use "AutoML for Image, Text, Time Series and Tabular Data", maybe "AutoML in 3 lines of code", etc.
- [x] Change code example to "train.csv", "test.csv", without functioning code, 3 lines, predict() instead of leaderboard()
- [x] Move most tutorials/papers/etc. to AWESOME.md, link to them in README.md
- [ ] Task Table: Rename first column to AutoGluon Predictor, Then introduce Task as the second column: Classification, Regression, TimeSeries: Time Series Forecasting, Multimodal: Images, Texts, Multimodal, Object Detection, Named Entity Recognition. Something short maybe: Image, Text, Multimodal. Highlight which tasks AutoGluon solves in 3 lines of code. Make Predictors clickable, potentially remove API column. Maybe a notes column with quick description.
- [ ] Add details collapsible headers for 3 LoC, Tabular is open, others are closed by default but can be opened. Question: Can we have buttons where only one details is visible at a time. Or maybe just have Tabular and leave others to tutorials.
- [x] Structure the links nicer
- [x] Update videos / talks
- [x] Update papers
- [x] Update cloud links 
- [x] Update articles (Remove old ones?) 
- [x] Include links to KDD/NeurIPS/etc. tutorials
- [ ] Performance comparisons
- [x] Move paper citations to a dedicated file and link to it
- [x] Cleanup tutorial links (Currently have a table and a bullet list, remove redundancy)
- [ ] Add discord / community section, maybe add to website too as a link

Maybe:

- [ ] Add feature checklist comparison with existing AutoML systems
- [ ] Enhance description of AutoGluon to highlight key ideas that achieve performance gains (highlighting ensembling and diversity maximization)
- [ ] Add image showing AutoMM Architecture (as in 1.0 release notes)
- [ ] Refactor code contribution guidelines

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,45 @@
...
-predictor = TabularPredictor(label='class').fit(train_data, time_limit=120)  # Fit models for 120s
-leaderboard = predictor.leaderboard(test_data)
+pip install autogluon
+```
+
+Visit our [Installation Guide](https://auto.gluon.ai/stable/install.html) for detailed instructions.
+
+## :zap: Quickstart
+
+Solve ML tasks end-to-end in just 3 lines of code!
```
Comment (shchur) [inline README.md:37]: What about "Build accurate end-to-end ML models in just 3 lines of code!"? I'm not sure what exactly solving ML tasks end-to-end means here.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
 
-[Install Instructions](https://auto.gluon.ai/stable/install.html) | Documentation ([Stable](https://auto.gluon.ai/stable/index.html) | [Latest](https://auto.gluon.ai/dev/index.html))
+[Install Instructions](https://auto.gluon.ai/stable/install.html) | [Documentation](https://auto.gluon.ai/stable/index.html) | [Release Notes](https://auto.gluon.ai/stable/whats_new/index.html)
 
 AutoGluon automates machine learning tasks enabling you to easily achieve strong predictive performance in your applications.  With just a few lines of code, you can train and deploy high-accuracy machine learning and deep learning models on image, text, time series, and tabular data.
+</div>
+
+## 💾 Installation
+
+AutoGluon is supported on Python 3.8 - 3.11 and is available on Linux, MacOS, and Windows OS.
```
Comment (shchur) [inline README.md:25]:
```suggestion
AutoGluon is supported on Python 3.8 - 3.11 and is available on Linux, MacOS, and Windows.
```

**Review by shchur — APPROVED** [review]
This looks amazing! Thank you @Innixma

File: CITING.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+BibTeX entry:
+
+```bibtex
+@inproceedings{agmultimodaltext,
+  title={Benchmarking Multimodal AutoML for Tabular Data with Text Fields},
+  author={Shi, Xingjian and Mueller, Jonas and Erickson, Nick and Li, Mu and Smola, Alexander J},
+  journal={Advances in Neural Information Processing Systems Datasets and Benchmarks Track},
+  volume={35},
+  year={2021}
+}
```
Comment (zhiqiangdon) [inline CITING.md:47]: This should be a temporary solution. We will replace this when the official AutoMM paper is available.

File: AWESOME.md [hunk 4]
```diff
@@ -8,8 +8,25 @@ We will be happy to add your success story using AutoGluon to this list.
...
+| :tv: [AutoGluon: re:MARS 2022 Keynote](https://www.youtube.com/watch?v=_wd0IJBTwbY&t=1277s)                                                                        | Keynote  | [Amazon re:MARS 2022](https://icml.cc/virtual/2020/workshop/5725)                                                                  | 2022/06/24 |
+| [Advancing the State of the Art in AutoML](https://developer.nvidia.com/blog/advancing-the-state-of-the-art-in-automl-now-10x-faster-with-nvidia-gpus-and-rapids/) | Talk     | [GTC 2021](https://developer.nvidia.com/blog/advancing-the-state-of-the-art-in-automl-now-10x-faster-with-nvidia-gpus-and-rapids/) | 2021/06/09 |
+| :tv: [AutoGluon and Distillation](https://icml.cc/virtual/2020/7045)                                                                                               | Keynote  | [ICML 2020, AutoML Workshop](https://icml.cc/virtual/2020/workshop/5725)                                                           | 2020/07/18 |
```
Comment (zhiqiangdon) [inline AWESOME.md:29]: Looks like that a comprehensive AutoMM tutorial is lacking. Though some tutorials mention AutoMM to some degree, their introductions are inconsistent with the current AutoMM story and shape. We can add one when available.

**Review by zhiqiangdon — APPROVED** [review]
LGTM. Left some comments which may be addressed in future.

File: CITING.md (hunk 3 above)
Comment (Innixma) [reply CITING.md:47]: Definitely! Once it is available, you can send a PR to update this section

File: AWESOME.md (hunk 4 above)
Comment (Innixma) [reply AWESOME.md:29]: If any talk we have given is missing, let me know and we can add it. If we need to do one, an option is for you to record it and we can upload it to YouTube, I created an AutoGluon YouTube channel.

File: README.md (hunk 2 above)
Comment (Innixma) [reply README.md]: Good catch, updated

File: README.md (hunk 1 above)
Comment (Innixma) [reply README.md]: Nice idea! Updated to this wording

**Pull request merged**